internal/util/fsql/tx.go

```diff
@@ -86,13 +91,17 @@ func (tx *Tx) QueryRowContext(ctx context.Context, query string, args ...any) *s
 
 	start := time.Now()
 
-	fields := []any{zap.Any("args", args)}
-	tx.l.Sugar().With(fields...).Debugf(">>> %s", query)
+	fields := []any{slog.Any("args", args)}
+	if tx.l.Enabled(ctx, slog.LevelDebug) {
+		tx.l.With(fields...).DebugContext(ctx, fmt.Sprintf(">>> %s", query))
```

Comment:
That check is not needed: `DebugContext` does nothing if debug level is not enabled

Reply:
I placed the check because evaluating `fmt.Sprintf` on query may be undesirable since `fmt.Sprintf` is evaluated (and not used) if log level set larger than debug. I check code path and `query` can't be too big. The largest possible is `ExecContext` used by insert, but still not as big as I initially thought.

It might look like below for inserting 15 docs.
`>>> INSERT INTO whateverthetable (_ferretdb_sjson) VALUES (?), (?), (?), (?), (?), (?), (?), (?), (?), (?), (?), (?), (?), (?)...` 

Well let me simplify to remove `Enabled(ctx, slog.LevelDebug)` check everywhere